### ISSUE TITLE
Unable to scrape URI 

### ISSUE THREAD

**Issue by philipschilling** [TryGhost/migrate#414]
When I run the command `migrate medium my-export-file.zip` I get many `Unable to scrape URI` errors for images such as:

`1997/2862: posts: are-there-any-gauchos-left-in-argentina feature_image - Unable to scrape URI https://cdn-images-1.medium.com/max/2560/1*utPlKXF5t00s2isvG7SZZw.jpeg
Total: 2862. Complete: 2541, Failed: 319, Skipped: 0, Disabled: 0`

However in my browser the image URLs are accessible. Furthermore after the statement `Write Ghost import zip` nothing happens for a while and at some point it states `client_loop: send disconnect: Broken pipe`

I am not sure if I am doing something wrong or if the migration script is broken. I do not use a custom domain. I have 166 blog posts which I want to migrate. When I first tried with less blog posts (I think less than 100), I also got the errors but at least a zip file was created on the server which I was able to use to import into Ghost. Right now the script does not create any zip archive on my server.

**Comment by PaulAdamDavis** [comment]
Hey @philipschilling I'm not able to replicate this locally with a few different Medium exports, both on an Intel and M1 Macs. 

It sounds like you're using this on a remote server? While the tool _can_ be used this way, it's primarily developed to be used locally. Would you be able install locally and see if that works better for you?

**Comment by philipschilling** [comment]
I indeed run it remotely on Digital Ocean. I noticed that the zip archive is generated when I reduce the number of blog posts to 50 or below. Maybe the server has some time limit. So for me it is fine now. I don't use a local development environment.

Just for your info what I noticed as well:
- The alt tag and caption of the feature image is not migrated
- The meta description and tags of the blog posts are not migrated
- Embedded elements such as Google Maps or YouTube / Vimeo videos are not properly migrated

Maybe this is also due to not having run the script properly. I split my project, did a basic migration and fix now everything manually which takes some time but is just a one-time job.
